src/moonstone-samples/lib/All/All.js

```diff
@@ -160,7 +160,7 @@ module.exports = kind({
 					{kind: Button, content: 'Back', small: true, href: 'index.html', mixins: [LinkSupport]}
 				],
 				components: [
-					{name: 'list', kind: DataList, fixedChildSize: 63, components: [
+					{name: 'list', kind: DataList, fixedChildSize: 62, components: [
```

Comment:
It's a shame we don't have an option to just set this value after measuring the height of the first element.